docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -0,0 +1,117 @@
+# List of Usability Tasks
+
+## Users
+
+As a user, you are a member of the general public who is using the Kukarella web application; you are someone who would
+like to have a more immersive text-to-speech experience where an animated avatar reads user text aloud.
+
+In your pursuit of having an animated text-to-speech experience, you will use our team's premium Kukarella account, which was provided by the client.
+
+### User Scenario 1: Language Test
+
+1. On the Kukarella web app
+
+    1. Enter text in a language other than English (you may use Google Translate, but your feedback will be more
+    valuable if you are familiar with the language)
+    2. Choose a voice for that language
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different language each time.
```

Comment:
How many times should we be repeating this? I would imagine we only want to do this once per participant?

Reply:
Good question. We can quantify this, for sure, but I was thinking at least twice so that the user can choose at least one other language

Reply:
We can hash out the details next-next Monday 😄